client/python/test/conftest.py

```diff
@@ -1,4 +1,159 @@
+import random
+import time
+
+import gradio as gr
+import pytest
+
+
 def pytest_configure(config):
     config.addinivalue_line(
         "markers", "flaky: mark test as flaky. Failure will not cause te"
     )
+
+
+@pytest.fixture
+def calculator():
+    def calculator(num1, operation, num2):
+        if operation == "add":
+            return num1 + num2
+        elif operation == "subtract":
+            return num1 - num2
+        elif operation == "multiply":
+            return num1 * num2
+        elif operation == "divide":
+            if num2 == 0:
+                raise gr.Error("Cannot divide by zero!")
+            return num1 / num2
+
+    demo = gr.Interface(
+        calculator,
+        ["number", gr.Radio(["add", "subtract", "multiply", "divide"]), "number"],
+        "number",
+        examples=[
+            [5, "add", 3],
+            [4, "divide", 2],
+            [-4, "multiply", 2.5],
+            [0, "subtract", 1.2],
+        ],
+    )
+    return demo.queue()
+
+
+@pytest.fixture
+def increment():
+    with gr.Blocks() as demo:
+        btn1 = gr.Button("Increment")
+        btn2 = gr.Button("Increment")
+        numb = gr.Number()
+
+        state = gr.State(0)
+
+        btn1.click(
+            lambda x: (x + 1, x + 1),
+            state,
+            [state, numb],
+            api_name="increment_with_queue",
+        )
+        btn2.click(
+            lambda x: (x + 1, x + 1),
+            state,
+            [state, numb],
+            queue=False,
+            api_name="increment_without_queue",
+        )
+    return demo.queue()
+
+
+@pytest.fixture
+def progress():
+    def my_function(x, progress=gr.Progress()):
+        progress(0, desc="Starting...")
+        for _ in progress.tqdm(range(20)):
+            time.sleep(0.1)
+        return x
+
+    return gr.Interface(my_function, gr.Textbox(), gr.Textbox()).queue()
+
+
+@pytest.fixture
+def yield_demo():
```

Comment:
Nit: for the sake of consistency, can we name all of the demo fixtures `*_demo()` like this one?